defusedxml/common.py

```diff
@@ -75,13 +75,25 @@ class NotSupportedError(DefusedXmlException):
 def _apply_defusing(defused_mod):
     assert defused_mod is sys.modules[defused_mod.__name__]
     stdlib_name = defused_mod.__origin__
-    __import__(stdlib_name, {}, {}, ["*"])
-    stdlib_mod = sys.modules[stdlib_name]
-    stdlib_names = set(dir(stdlib_mod))
-    for name, obj in vars(defused_mod).items():
-        if name.startswith("_") or name not in stdlib_names:
-            continue
-        setattr(stdlib_mod, name, obj)
+    if PY3:
```

Comment:
Is legacy Python still supported?!? https://github.com/tiran/defusedxml/blob/main/setup.py#L55